src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/ViewModels/ActionBarContextItemViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace Microsoft.CmdPal.UI.ViewModels;
+
+public partial class ActionBarContextItemViewModel : ObservableObject
+{
+    ////private ICommand _command;
+
+    [ObservableProperty]
+    private string _name = "Placeholder"; // Command.Name;
+
+    ////private IconDataType Icon => Command.Icon;
+
+    [ObservableProperty]
+    private bool _canInvoke = true; // Command != null && Command is IInvokableCommand or IPage;
+
+    // TODO: do we want the icon here or get it over in the UI project?
```

Comment:
Yeah, for the future we'll want to look at using converters and such for this and have an icon cache or something in the View layer.

Was seeing this with the 'TagViewModel' and brushes and such too, we'll just want to have a data template for tags in the UI layer for the collection that hosts them.